server/topic_page_config.textproto

```diff
@@ -0,0 +1,98 @@
+# Copyright 2021 Google LLC
```

Comment:
we'll have a set of these, one per topic x country, so would be nice to move them into a separate folder as we add more config files.